src/System.Security.Cryptography.Encoding/src/Internal/Cryptography/OidLookup.cs

```diff
@@ -108,7 +107,7 @@ public static string ToOid(string friendlyName, OidGroup oidGroup, bool fallBack
         // is to prevent issues wherein an identifier is different between CoreFX\Windows and CoreFX\Unix;
         // since any existing code would be using the Windows identifier, it is the de facto standard.
         private static readonly Dictionary<string, string> s_friendlyNameToOid =
-            new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+            new Dictionary<string, string>(103, StringComparer.OrdinalIgnoreCase)
```

Comment:
where does "103" come from?

Reply:
The number of items being added immediately after this line.  This is just pre-sizing it.